decidim-dev/lib/tasks/generators.rake

```diff
@@ -52,4 +53,14 @@ namespace :decidim do
       )
     end
   end
+
+  desc "Generates VAPID keys for push notifications"
+  task :generate do
```

Comment:
Ups - I've just seen that I didn't add any namespace here in the original AC, my bad!  `decidim:generate` is an awful naming. 

What about `decidim:webpush:generate_keys`? Or if you have a better proposal, feel free to share it @quinHD @ferblape

Reply:
Let's go with `gecidim:generate_vapid_keys`!

Reply:
```suggestion
  task :generate_vapid_keys do
```